app/views/RoomView/index.tsx

```diff
@@ -468,13 +512,13 @@ class RoomView extends React.Component<IRoomViewProps, IRoomViewState> {
 		}
 
 		let numIconsRight = 2;
-		if (tmid) {
+		if (tmid || (status && joined)) {
 			numIconsRight = 1;
 		} else if (teamId && isTeamRoom({ teamId, joined })) {
 			numIconsRight = 3;
 		}
 		const headerTitlePosition = getHeaderTitlePosition({ insets, numIconsRight });
-
+		const omnichannelPermissions = [canForwardGuest, canReturnQueue];
```

Comment:
```suggestion
		const omnichannelPermissions = {canForwardGuest, canReturnQueue};
```
With this change you can check via prop and not via index.

`omnichannelPermissions.canForwardGuest` =/= `omnichannelPermissions[0]`